pyuvdata/uvh5.py

```diff
@@ -458,8 +459,10 @@ def write_uvh5(self, filename, run_check=True, check_extra=True,
         try:
             import h5py
         except ImportError:  # pragma: no cover
-            raise ImportError('h5py is not installed but is required for '
-                              'uvh5 functionality')
+            from . import reraise_context
+            reraise_context('h5py is not installed but is required for '
```

Comment:
yes, that was an oversight. I originally put the function in `__init__.py` rather than utils, then moved it but failed to update all the right places.